tests/testthat/test-prepend.R

```diff
@@ -13,3 +13,27 @@ test_that("prepend is clearer version of merging with c()", {
     }
   )
 })
+
+test_that("prepend appends at the beginning for empty list by default", {
+  x <- list()
+  expect_identical(
+    x %>% prepend(1),
+    x %>% c(1, .)
+  )
+})
+
+test_that("prepend throws error if before param is neither NULL nor between 1 and length(x)", {
+  expect_error(
+    prepend(list(), 1, before = 1),
+    "is.null(before) || (before > 0 && before <= n) is not TRUE"
```

Comment:
This message testing seems too stringent because the message is generated upstream (by `base::stopifnot()`) and so is not under the control of purrr. Upstream changes are unlikely but would cause these tests to fail.